lib/pdn/pdn.c

```diff
@@ -0,0 +1,763 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include "pdn.h"
+#include "at_cmd.h"
+#include "at_notif.h"
+
+#include <errno.h>
+#include <string.h>
+#include <stdio.h>
+#include <net/socket.h>
+
+/**< Maximum buffer size. */
+#define PDN_AT_BUFFER_SIZE 256
```

Comment:
We need a comment explaining how this size was determined.

Reply:
I will add a comment.